src/containers/Tenant/Diagnostics/Overview/TableInfo/TableInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useMemo} from 'react';
+import cn from 'bem-cn-lite';
+
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+
+import {InfoViewer} from '../../../../../components/InfoViewer';
+
+import {getEntityName} from '../../../utils';
+
+import {prepareTableInfo} from '../utils';
+
+import '../Overview.scss';
+
+const b = cn('ydb-diagnostics-overview');
+
+interface TableInfoProps {
+    data?: TEvDescribeSchemeResult;
+}
+
+export const TableInfo = ({data}: TableInfoProps) => {
+    const entityName = getEntityName(data?.PathDescription);
+
+    const {
+        generalTableInfo = [],
+        tableStatsInfo = [],
+        tabletMetricsInfo = [],
+        partitionConfigInfo = [],
+    } = useMemo(() => prepareTableInfo(data), [data]);
+
+    if (
+        [generalTableInfo, tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat()
+            .length === 0
+    ) {
+        return <div className={b('title')}>{entityName}</div>;
+    }
+
+    return (
+        <div className={b()}>
+            <InfoViewer info={generalTableInfo} title={entityName} className={b('info-block')} />
+            <div className={b('row')}>
+                {tabletMetricsInfo.length > 0 || partitionConfigInfo.length > 0 ? (
+                    <div className={b('col')}>
+                        <InfoViewer
+                            info={tabletMetricsInfo}
+                            title={'Tablet Metrics'}
+                            className={b('info-block')}
+                            nullOnEmptyInfo
```

Comment:
Previously TableInfo (ex. SchemaInfoViewer) had function helper `renderItem`, that returned null on empty info. I decided to replace it with a new prop